### PR TITLE
Update to chroma-js v1.1.1

### PR DESCRIPTION
**Pull request by mpacholec**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

**Comment by invliD** [comment]
It appears you have removed a lot of documentation from the comments that I have added. Also some of the comments are formatted all wrong, it's

    /**
     * lorem ipsum
     */

instead of

    /**
    *
    */

(notice the missing indentation).

And please don't remove me as the author of the file. I did create it and a lot of what remains is still my work. You can add yourself as a second author if you want.
:-1:

File: chroma-js/chroma-js.d.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
-// Type definitions for Chroma.js v0.5.6
+// Type definitions for Chroma.js v1.1.1
 // Project: https://github.com/gka/chroma.js
-// Definitions by: Sebastian Brückner <https://github.com/invliD>
+// Definitions by: Marcin Pacholec <https://github.com/mpacholec>
```
Comment (mhegazy) [inline chroma-js/chroma-js.d.ts]: please add authors and do not remove existing ones.

File: chroma-js/chroma-js.d.ts [hunk 2]
```diff
@@ -101,217 +60,278 @@ declare namespace Chroma {
          */
         hex(color: string): Color;
 
-        rgb(red: number, green: number, blue: number, alpha?: number): Color;
-        hsl(hue: number, saturation: number, lightness: number, alpha?: number): Color;
-        hsv(hue: number, saturation: number, value: number, alpha?: number): Color;
+        /** */
```
Comment (mhegazy) [inline chroma-js/chroma-js.d.ts]: please remove empty comments

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**